indexer.py

```diff
@@ -149,10 +157,17 @@ def main():
         required=True,
         help="Name this experiment. We will store the artifacts under assets/<experiment-name>",
     )
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument(
+        "--function", action="store_true", help="Index at function granularity"
+    )
+    group.add_argument(
+        "--file", action="store_true", help="Index at file granularity"
+    )
     args = parser.parse_args()
 
     start = time.time()
-    indexer = Indexer(args.experiment_name)
+    indexer = Indexer(args.experiment_name, file=args.file)
```

Comment:
--function doesnt do anything except in the parser, maybe i get rid of the option and just make it the default behavior?